start new editors post (#255)
ropensci/roweb3: content/blog/2021-10-12-editors2021/index.md

---
title: 'Announcing New Software Peer Review Editors: Emily Riederer, Adam Sparks, and ... Bueller?'
author: Stefanie Butland

> date: '2021-10-27'

Author: ```suggestion
date: '2021-10-12'
```

# twitterImg: blog/2019/06/04/post-template/name-of-image.png
---
We are excited to welcome Emily Riederer, Adam Sparks, and Ferris Bueller to our team of [Associate Editors](/software-review/#editors) for rOpenSci [Software Peer Review](/software-review/). 
They join Laura DeCicco, Julia Gustavsen, Anna Krystalli, Mauro Lepore, Karthik Ram,Noam Ross, Maëlle Salmon, and Melina Vidoni. 

rOpenSci has been running a rigorous and collegial software peer review system since 2015. 
Editors [manage the review process](https://devguide.ropensci.org/editorguide.html), performing initial package checks, identifying reviewers, and moderating the process until the package is accepted by reviewers and transferred to rOpenSci. 
To address the scope and volume of packages submitted for review, it's critical that we have a team of editors with complementary expertise.
Adam brings his broad experience with API packages and his experience from agricultural research in academia and government.
Emily is known for openly sharing her processes-oriented approach to development based in her experience in industry.

Meet our new editors!

## Emily Riederer

> {{< imgtxt src = "emily-riederer.jpg" width = "200" alt = "headshot of Emily Riederer">}} 

Author: ```suggestion
{{< imgtxt src = "/img/team/emily-riederer.jpg" width = "200" alt = "headshot of Emily Riederer">}} 
```

Emily reviewed the [tradestatistics](https://github.com/ropensci/software-review/issues/274) package for rOpenSci, has guest-edited the review of [dataspice](https://github.com/ropensci/software-review/issues/426) and is currently handling [cffr](https://github.com/ropensci/software-review/issues/463) for generating Citation File Format ('cff') metadata for R Packages.

{{< /imgtxt >}}

Emily on [GitHub](https://github.com/emilyriederer), [Twitter](https://twitter.com/emilyriederer), [Website](https://emilyriederer.netlify.com/)

> > I had the great fortune of stumbling upon rOpenSci relatively earliy in my career as an R programmer, and it's not exaggeration to say that this organization has had one of the single largest impacts on my career. rOpenSci's meticulous documentation helped me to embrace best packages in my development work. Perhaps even more impactful is how watching rOpenSci work shaped my perspective on community building, leadership, and teamwork. I have always admired the well-oiled machine that rOpenSci has created -- sustaining massive value with limited individual overhead by clearly defining expectations, providing opinionated infrastrcutre, and cultivating a phenomenal communtiy. I'm thrilled to be taking a formal role in an organization that has had such an impact on me. 

Author: ```suggestion
> I had the great fortune of stumbling upon rOpenSci relatively early in my career as an R programmer, and it's not an exaggeration to say that this organization has had one of the single largest impacts on my career. rOpenSci's meticulous documentation helped me to embrace best practices in my development work. Perhaps even more impactful is how watching rOpenSci work shaped my perspective on community building, leadership, and teamwork. I have always admired the well-oiled machine that rOpenSci has created -- sustaining massive value with limited individual overhead by clearly defining expectations, providing opinionated infrastructure, and cultivating a phenomenal community. I'm thrilled to be taking a formal role in an organization that has had such an impact on me. 
```

Author: @emilyriederer I changed "embrace best packages" to "embrace best practices"
Is that what you meant?

Thanks so much for the generous quote.

Reviewer: Oh yikes - thank you, @stefaniebutland ! That is definitely what I meant. I appreciate the typo fixes also. I wish my editor announcement hadn't needed so much editing 🙂
